client/src/components/ManageAccount/ManageAccountTables.jsx

```diff
@@ -82,18 +81,18 @@ export const RequiredInformation = () => {
       manualHeader={
         <SectionHeader
           actions={
-            <Button
-              type="link"
-              className="form-button"
-              onClick={openModal}
-              disabled={errors.fields !== undefined}
+            <a
+              className={`wb-link ${styles['edit-profile-link']}`}
+              href="https://accounts.tacc.utexas.edu/profile"
```

Comment:
Actually, I wonder if hardcoding the link to `accounts.tacc.utexas.edu` will be good long term, since there is the possibility portals may have a custom TAM site built for them, for example apcd.

We could implement something similar to this pr to make it dynamic: https://github.com/TACC/Core-Portal/pull/688

Reply:
sounds good. will reach out to you to discuss how it should look in WORKBENCH_SETTINGS

Reply:
Follow on ticket should be fine I think

Reply:
Ticket added:  https://jira.tacc.utexas.edu/browse/FP-2021